docs/csharp/programming-guide/classes-and-structs/extension-methods.md

```diff
@@ -24,13 +24,13 @@ Extension methods enable you to "add" methods to existing types without creating
   
  The `WordCount` extension method can be brought into scope with this `using` directive:  
   
-```  
+```xml  
```

Comment:
That one should be `csharp`, shouldn't it?

Reply:
Thanks @pkulikov! I did these with a tool and reviewed by hand, but knew I'd miss one! 😩